### PR TITLE
DC-03 News Blog

### PR DESCRIPTION
**Pull request by mayurdawnge45**
Hi @akash-dhande 
Ticket: https://dhandecreative.atlassian.net/browse/DC-3

- News blog section has been completed.

Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -141,6 +141,42 @@ <h2>Happy Client</h2>
...
+                </i>
+                <i class="fab fa-facebook">
+                    <p>facebook</p>
+                </i>
+            </div>
+        </div>
+        <div class="mask-group">
+            <div class="left-maskgroup"><img src="assets/images/Mask Group.png" alt=""></div>
+            <div class="center-maskgroup"><img src="assets/images/4188214 1.png" alt=""></div>
+            <div class="right-maskgroup"><img src="assets/images/53 1817143482.png" alt=""></div>
```
Comment (akash-dhande) [inline index.html]: File naming is wrong

**Review by akash-dhande — CHANGES_REQUESTED** [review]
small change

**Comment by akash-dhande** [comment]
Keep consistency in PR title "Dc 03 news blog" > "DC-03 News Blog"

**Comment by akash-dhande** [comment]
Add ticket url in PR details

**Comment by akash-dhande** [comment]
Ticket: [TICKET-URL]
- Description detail 01
- Description detail 02
  - Description sub detail 02.01
- Description detail 03

Thanks

**Comment by akash-dhande** [comment]
> * ription detail 01
> * Description detail 02
>   
>   * Description sub detail 02.01
> * Description detail 03
> 
> Thanks

Updated the description.

File: index.html (hunk 1 above)
Comment (mayurdawnge45) [reply index.html]: Hi @akash-dhande
https://dhandecreative.atlassian.net/browse/DC-3 
I have changed the file name and its Dependences.

Thanks.

**Review by akash-dhande — APPROVED** [review]

**Pull request merged**